src/Homepage/index.js

```diff
@@ -367,7 +416,13 @@ class Homepage extends React.Component {
       lng: Number(res.lng),
     }));
     const { translations } = this.props;
-    return (
+    return this.state.showPageLoader ? (
+      <div className="text-center py-5">
+        <div className="spinner-border text-secondary" role="status">
+          <span className="sr-only">Loading...</span>
+        </div>
+      </div>
```

Comment:
Maybe we can move this component to App level as well. And pass the `togglePageLoader` method in global context to all its child components. @arjun27 what do you think ?

Reply:
If we move this up to App level, what would be part of render when the page is loading?

Reply:
I was thinking to be something like this. So, that other routes can access and toggle it. 

<img width="790" alt="Screenshot 2020-05-04 at 9 17 28 AM" src="https://user-images.githubusercontent.com/5427715/80943713-1d328280-8de8-11ea-944f-58c727331349.png">

Reply:
But this is not super necessary right now